docs/getting-started/using-isis-first-steps/introduction-to-isis.md

```diff
@@ -0,0 +1,373 @@
+# Introduction to ISIS
+
+## What is ISIS?
+
+-----
+
+ISIS is an image processing software package. The focus of the software
+is to manipulate imagery collected by current and past NASA planetary
+missions sent to Mars, Jupiter, Saturn, and other solar system bodies.
+
+If you're familiar with commercial image processing packages like 
+Photoshop, Envi, or ERDAS Imagine, you'll recognize many of the 
+standard image processing operations ISIS is capable of, such as 
+contrast, stretch, image algebra, filters, and statistical analysis.
+
+However, the real power of ISIS is its unique capabilities for
+processing data from NASA spacecraft missions such as Voyager, Viking,
+Galileo, Mars Global Surveyor, and Mars Odyssey. ISIS is able to import
+raw mission data into a usable geospatial image product, and has tools
+for digital mosaicking of adjacent images, photometric modeling and
+normalization, removal of systematic noise patterns, overlaying
+graticules, and numerous other cartographic and scientific analysis
+functions.
+
+!!! info inline end "Valles Marineris Mosaic"
+
+    ![Mars Hemisphere Mosaic](../../assets/isis-fundamentals/Mars_hemi.gif){ width=200 }
+
+    This famous mosaic of the Valles Marineris hemisphere of Mars was
+    created using ISIS. The mosaic is composed of 102 Viking Orbiter
+    images of Mars, and is projected into point perspective, a view
+    similar to that which one would see from a spacecraft at a distance of
+    2500 kilometers from the surface of the planet.
+
+The following are just a few examples of how the USGS Astrogeology
+Research Program has used ISIS in our work (links open in a new
+window):
+
+  - **Global mosaics** : mosaicking hundreds or thousands of images
+    collected by space exploration missions to create seamless,
+    cartographically accurate, global image maps for use by the
+    planetary science community for research and mapping. See our
+    [Map-a-Planet](https://astrogeology.usgs.gov/tools/map-a-planet-2) site to view these
+    products.
+
+  - **Geologic Mapping** : we create accurate base image maps for
+    geologists to use in creating geologic maps. See our [Planetary
+    Geologic Mapping
+    Program](http://astrogeology.usgs.gov/Projects/PlanetaryMapping/)
+    for more information about this work.
+
+  - **Scientific Research** : Using ISIS to mosaic images of a region
+    of interest to create a scientifically accurate image product, and
+    analyzing the imagery based on spectral, textural, or other
+    attributes. See the following Science Magazine abstract for examples
+    of images from the Mars Exploration Rover Mission Microscopic Image
+    that were processed and analyzed using ISIS as part of the
+    scientific research resulting from the mission: Textures of the
+    [Soils and Rocks at Gusev Crater from Spirit's Microscopic
```

Comment:
Since the paper name is "Textures of the Soils and Rocks at Gusev Crater from Spirit’s Microscopic Imager", "Textures of the" should be part of the hyperlink.